connectors/concur/src/main/resources/application.yml

```diff
@@ -3,3 +3,6 @@ server:
      enabled: true
      mime-types: application/json,application/xml,text/html,text/xml,text/plain
   server-header: concur-connector
+
+connector:
+  hasTestAuth: true
```

Comment:
If Concur is using OAuth (not Http Basic Auth), then you shouldn't need a test-auth endpoint.  The test-auth endpoint is only useful because the client does not know whether the user typed in a bad user/password when prompted for Basic credentials.

With OAuth, the user is potentially prompted for user/password on the third party's website and given a screen to click Allow/Deny.  If the user messed up their credentials with OAuth, they would know about it immediately.

With Basic, we don't know if the credentials are good until the client tries to use them.  This is why the test-auth endpoint exists, so the client can check immediately instead of finding out when /requests/cards is called into.